quickstarts/microsoft.machinelearningservices/machine-learning-end-to-end-secure/modules/keyvault.bicep

```diff
@@ -1,5 +1,5 @@
 // Creates a KeyVault with Private Link Endpoint
-@description('The Azure Region to deploy the resrouce group into')
+@description('The Azure Region to deploy the resource into')
```

Comment:
should be ```resources``` - also you need to compile the bicep and also submit the changed azuredeploy.json

Reply:
updated.  When I ran a bicep build it changed more than just the spelling, seemed to remove some endpoint version info.  I'm not clear if this was previously built.

For this PR I have just fixed the spelling issue.

Reply:
Example dropped api version after a bicep build

![image](https://user-images.githubusercontent.com/7253192/154391358-e342bb9f-06dd-410c-ad68-0cc8d03ea79d.png)